package.json

```diff
@@ -0,0 +1,8 @@
+{
+  "private": true,
+  "workspaces": ["saturday-afternoon"],
+  "scripts": {
+    "eslint": "eslint .",
+    "test-ci": "yarn eslint && yarn flow"
```

Comment:
I'd prefer having `flow check` as a script defined in this file.

Reply:
fixed